src/Symfony/Bridge/Doctrine/DependencyInjection/CompilerPass/RegisterEventListenersAndSubscribersPass.php

```diff
@@ -66,6 +66,10 @@ public function process(ContainerBuilder $container)
             $a = isset($a['priority']) ? $a['priority'] : 0;
             $b = isset($b['priority']) ? $b['priority'] : 0;
 
+            if ($a === $b) {
+                return 0;
```

Comment:
Let's return $b - $a and remove both the if and the ternary operator.

Reply:
done!